packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,153 @@
+:local {
+  .checkbox {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: block;
+    font-family: inherit;
+    margin-left: var(--terra-form-checkbox-container-margin-left, none);
+    margin-right: var(--terra-form-checkbox-container-margin-right, 2rem);
+    margin-top: var(--terra-form-checkbox-container-margin-top, 0.625rem);
+
+    /* Hitbox and container focus style will only surround checkbox + label, instead of the div block */
+    /* Padding puts space for container focus style */
+    .inner-div {
+      display: inline-block;
+      padding-left: var(--terra-form-checkbox-inner-div-padding-left, 0.214rem);
+      padding-right: var(--terra-form-checkboc-innter-div-padding-right, 0.214rem);
+    }
+
+    /* Hide label if isLabelHidden true */
+    .label-hidden {
+      clip: rect(0 0 0 0);
+      cursor: default;
+      visibility: hidden;
+    }
+
+    /* stylelint-disable selector-max-compound-selectors */
+    .label {
+      cursor: pointer;
+      display: flex;
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+      line-height: 1.42em;
+      padding-left: var(--terra-form-checkbox-padding-left, none);
+      position: relative;
+    }
+
+    /* Native checkbox */
```

Comment:
why do we define styles for native and custom checkbox? Which browser native indicate here?

Reply:
Most browsers display the `input` element with browser native styling, no matter what styles you apply. So, for the themes that utilize browser native styling,`input` gets displayed. On themes that want custom checkbox styling, `input` gets hidden (but is still access-able), and the checkbox styling gets applied to a css checkbox next to the label text.

Reply:
Could you add an comment indicating this behavior?